KarhooUISDK/Screens/BookingRequestScreen/FormBookingRequestScreen/FormBookingRequestViewController.swift

```diff
@@ -118,129 +225,154 @@ final class FormBookingRequestViewController: UIViewController, BookingRequestVi
-                                                              constant: textInputInset),
-                 poiDetailsInputText.trailingAnchor.constraint(equalTo: baseStackView.trailingAnchor,
-                                                               constant: -textInputInset)].map { $0.isActive = true }
-            
-            _ = [commentsInputText.leadingAnchor.constraint(equalTo: baseStackView.leadingAnchor,
-                                                            constant: textInputInset),
-                 commentsInputText.trailingAnchor.constraint(equalTo: baseStackView.trailingAnchor,
-                                                             constant: -textInputInset)].map { $0.isActive = true }
-            
-            _ = [paymentDetailsTitle.leadingAnchor.constraint(equalTo: baseStackView.leadingAnchor,
-                                                              constant: titleInset),
-                 paymentDetailsTitle.trailingAnchor.constraint(equalTo: baseStackView.trailingAnchor,
-                                                               constant: -titleInset)].map { $0.isActive = true }
-            
-            _ = [addPaymentView.leadingAnchor.constraint(equalTo: baseStackView.leadingAnchor,
-                                                         constant: textInputInset),
-                 addPaymentView.trailingAnchor.constraint(equalTo: baseStackView.trailingAnchor,
-                                                          constant: -textInputInset)].map { $0.isActive = true }
-            
-            _ = [footerView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 10.0),
-                 footerView.trailingAnchor.constraint(equalTo: view.trailingAnchor,
-                                                      constant: -10.0)].map { $0.isActive = true }
-            
-            _ = [footerStack.topAnchor.constraint(equalTo: footerView.topAnchor),
-                 footerStack.leadingAnchor.constraint(equalTo: footerView.leadingAnchor),
-                 footerStack.trailingAnchor.constraint(equalTo: footerView.trailingAnchor),
-                 footerStack.bottomAnchor.constraint(equalTo: footerView.bottomAnchor)].map { $0.isActive = true }
-            
-            _ = [termsConditionsView.leadingAnchor.constraint(equalTo: footerStack.leadingAnchor),
-                 termsConditionsView.trailingAnchor.constraint(equalTo: footerStack.trailingAnchor)]
-                .map { $0.isActive = true }
-            
-            _ = [separatorLine.heightAnchor.constraint(equalToConstant: 1.0),
-                 separatorLine.leadingAnchor.constraint(equalTo: footerStack.leadingAnchor),
-                 separatorLine.trailingAnchor.constraint(equalTo: footerStack.trailingAnchor)]
+                 container.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.width)]
                 .map { $0.isActive = true }
             
+            if presenter.isKarhooUser() {
```

Comment:
I dont think we need the logic to separate the KarhooUser from other auth methods ... you can use `setupConstraintsForDefault()` for both of the cases here

Reply:
I added it only for hiding/showing some UI elements